src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -1210,6 +1215,19 @@ public static function closeOutputBuffers(int $targetLevel, bool $flush): void
         }
     }
 
+    /**
+     * Mark a response as safe according to RFC8674.
+     *
+     * @see https://tools.ietf.org/html/rfc8674
+     */
+    public function setSafe()
```

Comment:
`setSafeContent()`? better idea?

Reply:
Go for `setSafeContent()` until a better name shows up :slightly_smiling_face: